sandbox/kind/README.md

```diff
@@ -106,6 +106,16 @@ kubectl get secret "$secretname" --namespace=kubernetes-dashboard -o template --
 ```  
 - Open your browser at this address `http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/#/login`, copy/paste the token
   and enjoy your K8s experience
+  
+The script [kind-reg-ingress.sh](kind-reg-ingress.sh) allows to create a kind cluster with a local registry and install an Ingress controller in the cluster.
```

Comment:
I propose this wording `The bash script [kind-reg-ingress.sh](kind-reg-ingress.sh) allows to create a Kubernetes cluster using the tool -kind, integrated with a container private registry where an Ingress controller is deployed.`